src/object-inspector/ObjectPreview.js

```diff
@@ -36,15 +36,26 @@ const ObjectPreview = ({ data, maxProperties }) => {
   }
 
   if (Array.isArray(object)) {
+    let previewArray;
```

Comment:
You have 2 map implementations that are doing the same thing.  How would you feel about the following suggestion?

```suggestion
    const previewArray = object
      .slice(0, maxProperties)
      .map((element, index) => <ObjectValue key={index} object={element} />);
    if (object.length > maxProperties) {
      previewArray.push(<span key="ellipsis">…</span>);
    }
    
```

That way, if the map function ever gets more complex, we can update it in one place instead of 2.  Thoughts?

Reply:
oh yes that is much better, thanks @cgood92 !

Reply:
Hmm...... I'm sorry, I think I did you a disservice. I must have used the GitHub suggestion tool incorrectly, because when you accepted it, it now has duplicate lines and is not what was intended. I think the only way to fix this is if you pull it down and fix it locally. Can you work on that @bcolloran ? Again, I'm sorry because I think it was my fault.

Reply:
Heh no problem, I've never used it either, perhaps they're still working out the kinks... I now see the "beta" label there. :-) I'll take care of it

Reply:
okeedoke, hopefully we should be set now, thanks @cgood92 !

Reply:
Perfect.  Thanks @bcolloran.  I will merge, as I think @CodyReichert is ok with us doing, from his comments.